### PR TITLE
Steering.md

### PR DESCRIPTION
**Pull request by granvilj**
charter document for iTwin.js steering committee.

### CODE REVIEW

**Comment by calebmshafer** [comment]
@granvilj this looks good overall. However, let's get the announcement written up before merging this and making it official.

File: Steering.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+1. Be actively developing an application or solution using iTwin.js 
+2. Be willing and able to attend regularly scheduled Steering Committee meetings 
+
+Candidates for membership will be nominated by current Steering Committee members or by iTwin.js maintainers.  The addition of new members will be conducted by a standard motion and vote.  A member may be removed from the Steering Committee by voluntary resignation, by a standard committee motion and vote. 
+
+The Steering Committee will elect amongst its members a Chairperson to work with the iTwin.js maintainers to build an agenda for the meetings.  The committee shall hold annual elections to select a Chairperson for a term of one year.  There are no limits on the number of terms a Chairperson may serve.   
+
+## Meetings 
+
+Meetings will happen periodically not more than once a quarter. Recordings and minutes will be posted publicly.
```
Comment (pmconne) [inline Steering.md:28]: Where will meetings happen? Where will recordings and minutes be posted?

File: Steering.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# iTwin.js Steering Committee Charter
```
Comment (pmconne) [inline Steering.md:1]: Do you intend to link to this document from somewhere? Currently, no one will see it unless they happen to browse the source.

File: Steering.md (hunk 1 above)
Comment (calebmshafer) [reply Steering.md:28]: The meetings will either be on Teams or Zoom and a link will be provided. We're still figuring out where things will be posted but I would expect recordings to YouTube and Minutes either in a GH repo or in some public Word doc.

File: Steering.md (hunk 2 above)
Comment (calebmshafer) [reply Steering.md:1]: Yes, a blog post is going to be created and then promoted on various social media outlets including the Momentum event.

File: Steering.md (hunk 1 above)
Comment (granvilj) [reply Steering.md:28]: We can create a folder in this repo where meeting minutes can be recorded.

File: Steering.md (hunk 2 above)
Comment (granvilj) [reply Steering.md:1]: an announcement blog post is being crafted and will link to this charter document.

File: Steering.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+The Steering Committee’s responsibilities are to: 
+
+1. Provide feedback to the development team about the functionality, accessibility, and usability of iTwin.js. 
+2. Assist iTwin.js development team in prioritizing upcoming roadmap items and planned work. 
+3. Represent the "voice of the iTwin.js user" to the broader community. 
+
+## Membership 
+
+The Steering Committee memberships are not time-limited nor is there a maximum size limit.  The size of the committee is expected to vary to ensure adequate coverage in area expertise and diversity of types of users.  The Steering Committee must have a minimum of four members, and no more than one-fourth of the members maybe affiliated with the same employer.
```
Comment (wgoehrig) [inline Steering.md:15]:
```suggestion
The Steering Committee memberships are not time-limited nor is there a maximum size limit.  The size of the committee is expected to vary to ensure adequate coverage in area expertise and diversity of types of users.  The Steering Committee must have a minimum of four members, and no more than one-fourth of the members may be affiliated with the same employer.    
```

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**